### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by MarioDevment**
Update dependencies and kill some infections

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,29 +22,29 @@
...
     "require": {
         "php": ">= 7.2",
-        "linio/input": "^2.5",
+        "linio/input": "^2.7",
         "psr/container": "~1.0",
         "psr/http-message": "^1.0",
-        "psr/log": "~1.0",
-        "ramsey/uuid": "^3.8",
+        "psr/log": "~1.1",
+        "ramsey/uuid": "^3.9|^4.0",
```
Comment (paulotokimatu) [inline composer.json:29]: Is 3.9 necessary?

Comment (MarioDevment) [reply composer.json:29]: Just a precaution, we have not yet updated our APIs, I do not know what scope this change would have.

Comment (paulotokimatu) [reply composer.json:29]: I see, thanks!

Comment (bycosta) [reply composer.json:29]: Remember this library is public.
If the `4.x` version brings BC to this library we should just do a new release here bumping the major version.

Comment (MarioDevment) [reply composer.json:29]: I think if it would bring some BC.

https://uuid.ramsey.dev/en/latest/upgrading/3-to-4.html

Comment (bycosta) [reply composer.json:29]: I think what you did is ok! 😄

**Pull request merged**